src/EscolaLmsFilesServiceProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace EscolaLms\Files;
+
+use EscolaLms\Core\Providers\Injectable;
+use EscolaLms\Files\Http\Services\Contracts\FileServiceContract;
+use EscolaLms\Files\Http\Services\FileService;
+use Illuminate\Support\ServiceProvider;
+
+class EscolaLmsFilesServiceProvider extends ServiceProvider
+{
+    use Injectable;
```

Comment:
imho this is uselsess 

https://laravel.com/docs/8.x/providers#the-bindings-and-singletons-properties

example below, buildin Laravel Service provider resolves this 

```php
 public $bindings = [
        ServerProvider::class => DigitalOceanServerProvider::class,
    ];

 public $singletons = [
        DowntimeNotifier::class => PingdomDowntimeNotifier::class,
        ServerProvider::class => ServerToolsProvider::class,
 ];
```

i would prefer singletons over binds

Reply:
I don't quite understand. You mean to not use contracts, but singletons instead?

Reply:
`use EscolaLms\Core\Providers\Injectable;` does the same as builtin features, maybe it was making sense in L7 but in L8 does not.

Reply:
https://github.com/EscolaLMS/Core/issues/9